packages/cli/src/DebugInfo.ts

```diff
@@ -41,6 +51,10 @@ export class DebugInfo implements Command {
       return this.help()
     }
 
+    // Loads the `.env` file if it exists
+    // and prints a line to stdout to inform the users
+    loadEnvFile(undefined, true)
```

Comment:
I see `undefined` seems to be normal here and taken from `packages/cli/src/Version.ts`.

Reply:
Switching to `loadEnvFile(args['--schema'], true)`